data/org.freedesktop.impl.portal.RemoteDesktop.xml

```diff
@@ -39,6 +39,16 @@
         org.freedesktop.portal.ScreenCast), but may only be started and stopped
         with this interface.
 
+        The following results get passed via the options argument:
+        <variablelist>
+          <varlistentry>
+            <term>pid t</term>
+            <listitem><para>
+              The process id of the process that is creating the session.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
```

Comment:
I wonder if this couldn't instead be something that could be put on the `org.freedesktop.portal.Session` API instead.

Reply:
Erm, yes, I meant `org.freedesktop.impl.portal.Session`.

Reply:
Okay, If you all agree that this is the way to go, I'll implement this instead